classes/class-wc-rest-connect-services-controller.php

```diff
@@ -96,7 +96,10 @@ public function update_item( $request ) {
 		if ( empty( $settings ) ) {
 			$error = new WP_Error( 'bad_form_data',
 				__( 'Unable to update service settings. The form data could not be read.', 'woocommerce-services' ),
-				array( 'status' => 400 )
+				array(
+					'status'     => 400,
+					'error_code' => json_last_error(),
+				)
 			);
 			$this->logger->debug( $error, __CLASS__ );
```

Comment:
Two questions:
1) Is this worth logging whether or not debug is enabled?
2) Would it be worth surfacing as an admin notice, or at least re-use the existing notice that directs the user to the logs?